build/capabilities.json

```diff
@@ -1810,7 +1810,7 @@
           }
```

Comment:
This file is actually serialized from the built-in structs in OPA! See this file: https://github.com/open-policy-agent/opa/blob/main/ast/builtins.go

I'm happy to merge the PR with the changes here, but they'll be overwritten the next time we generate this file from OPA data + our two custom built-ins.

Can you consider fixing these typos in the OPA repo? I'm sure they'd appreciate it :)